models/pipeline.js

```diff
@@ -62,7 +67,7 @@ module.exports = {
      * @type {Joi}
      */
     get: Joi.object(mutate(MODEL, [
-        'id', 'scmUri', 'createTime', 'admins'
+        'id', 'scmUri', 'scmContext', 'createTime', 'admins'
```

Comment:
But that means it will be blocked by the sequelize team. We don't know when they would publish a new version. Would there be any harm in leaving it as `optional` then `required` later?